### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nilemarbarcelos**
Added an updated version of the response

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,5 +29,21 @@ curl http://localhost:3000/weather/Curitiba
 ```
 #### Response
 ```sh
-{"temperature":"14 °C","wind":"6 km/h"}
+{  
+   temperature:"24 °C",
```
Comment (robertoduessmann) [inline README.md]: Thanks for contribution @nilemarbarcelos :grinning: 
Just don't forget to update the json without the format (tags between quotation marks).

File: README.md [hunk 2]
```diff
@@ -29,5 +29,21 @@ curl http://localhost:3000/weather/Curitiba
 ```
 #### Response
 ```sh
```
Comment (robertoduessmann) [inline README.md]: If you can, please update also here to `json` to show formatted.

**Comment by nilemarbarcelos** [comment]
@robertoduessmann done 😄

**Review by robertoduessmann — APPROVED** [review]
Great @nilemarbarcelos , thanks :tada:

**Pull request merged**